packages/openstef-beam/src/openstef_beam/metrics/metrics_deterministic.py

```diff
@@ -345,3 +345,131 @@ def fbeta(
         return 0.0
 
     return (1 + beta**2) * (precision * recall) / (beta**2 * precision + recall)
+
+
+def riqd(
+    y_true: npt.NDArray[np.floating],
+    y_pred_lower_q: npt.NDArray[np.floating],
+    y_pred_upper_q: npt.NDArray[np.floating],
+    *,
+    measurement_range_lower_q: float = 0.05,
+    measurement_range_upper_q: float = 0.95,
+) -> float:
+    """Calculate the relative Inter Quantile Distance (rIQD).
+
+    rIQD measures the average distance between two quantiles, normalized by the measurement range.
+
+    Args:
+        y_true: Ground truth values with shape (num_samples,). Should not contain
+            values close to zero to avoid division issues.
```

Comment:
Copy-paste from mape I think, not really relevant here.
```suggestion
        y_true: Ground truth values with shape (num_samples,).
```